bip-0118.mediawiki

```diff
@@ -13,7 +13,8 @@
 
 == Abstract ==
 This BIP describes a new signature hash flag (<tt>sighash</tt>-flag) for
-segwit transactions. It removes any commitment to the output being
+segwit 
+ns. It removes any commitment to the output being
```

Comment:
This doesn't look like a typo fix...

Reply:
Yeah, sorry, the Github in-line editor messed it all up.
I've pushed a fixed version.